### DISCUSSION TITLE
Error When I run the command for docker compose

### DISCUSSION THREAD

**Discussion by edsongomes1** [aspnetrun/run-aspnetcore-microservices#35] [General]
Hi, thanks for the course. But I'm new with .net core, microservices ands dockers.  I try to run the command docker-compose -f docker-compose.yml -f docker-compose.override.yml up -d

time="2021-08-11T21:19:06-03:00" level=warning msg="The DOCKER_REGISTRY variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The DOCKER_REGISTRY variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The DOCKER_REGISTRY variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The WEBAPP_STORAGE_HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
time="2021-08-11T21:19:06-03:00" level=warning msg="The HOME variable is not set. Defaulting to a blank string."
[+] Running 4/7

Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /.aspnet/https

I can't run
Catalog API -> http://host.docker.internal:8000/swagger/index.html

Basket API -> http://host.docker.internal:8001/swagger/index.html

Discount API -> http://host.docker.internal:8002/swagger/index.html

Ordering API -> http://host.docker.internal:8004/swagger/index.html

Shopping.Aggregator -> http://host.docker.internal:8005/swagger/index.html

**Comment by nealu** [comment]
Same I cannot run and I get the error since redirect from host,docker,internal is not covered.

This site can’t be reachedhost.docker.internal’s server IP address could not be found.
Try:

Checking the connection
[Checking the proxy, firewall, and DNS configuration](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_NAME_NOT_RESOLVED
